src/tpl.c

```diff
@@ -229,7 +230,10 @@ tpl_hook_t tpl_hook = {
 };
 
 static const char tpl_fmt_chars[] = "AS($)BiucsfIUjv#p"; /* valid format chars */
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-variable"
 static const char tpl_S_fmt_chars[] = "iucsfIUjv#$()p"; /* valid within S(...) */
+#pragma clang diagnostic pop
```

Comment:
VS reports `warning C4068: Unbekanntes Pragma`, thus you probably shoudl wrap the pragma in #ifdef.